src/aap_eda/services/activation/manager.py

```diff
@@ -1041,6 +1041,17 @@ def _create_activation_instance(self):
             if hasattr(self.db_instance, "git_hash")
             else ""
         )
+        with transaction.atomic():
```

Comment:
We should place line 1063 https://github.com/ansible/eda-server/pull/675/files#diff-767f95d0b4918e197417276be62147629e68416233ba137c48b8c171b570f73cR1063 inside the transaction block.

Reply:
added.